src/git/repository.ts

```diff
@@ -74,7 +74,7 @@ export async function getGitHubUrls(): Promise<GitHubUrls[] | null> {
           remote = [r.state.remotes[0]];
         }
 
-        if (remote.length > 0 && remote[0].pushUrl?.indexOf("github.com") !== -1) {
+        if (remote.length > 0 && (remote[0].pushUrl?.indexOf("github.com") !== -1 || isUseEnterprise())) {
```

Comment:
Can we check if the remote contains the configured enterprise url?

Reply:
good idea, we'd need to get the domain only from the URL, since the whole thing includes the API endpoint for octokit requests. Fortunately Node has the URL class built in since v10.

Reply:
implemented by e6fa56779d7445e1ccbbce5cbbc3112922c1529c